mdn/tests/test_kumascript.py

```diff
@@ -31,6 +31,36 @@ def test_known(self):
         self.assertTrue(ks.known)
 
 
+class TestBug(TestCase):
+    # https://developer.mozilla.org/en-US/docs/Template:Bug
+    def test_plain(self):
+        # https://developer.mozilla.org/en-US/docs/Web/API/Node/isSupported
+        raw = '{{bug("801425")}}'
+        ks = Bug(raw=raw, args=['801425'], scope='footnote')
+        expected = (
+            '<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=801425">'
+            'bug 801425</a>')
+        self.assertEqual(ks.to_html(), expected)
+        self.assertEqual(ks.issues, [])
+
+    def test_summary(self):
+        # https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter
+        raw = "{{bug(1178765, 'summary')}}"
```

Comment:
This is the kind of test that seems superfluous to me if we already have a test that tests the `min_args` and `max_args` functionality?